scylla/src/statement/prepared_statement.rs

```diff
@@ -620,7 +621,7 @@ mod tests {
         let col_specs: Vec<_> = cols
             .into_iter()
             .enumerate()
-            .map(|(i, typ)| ColumnSpec::new_for_test(table_spec.clone(), format!("col_{}", i), typ))
+            .map(|(i, typ)| ColumnSpec::owned(format!("col_{}", i), typ, table_spec.clone()))
```

Comment:
Why we decided not to use `new_for_test` here anymore? I actually like `ColumnSpec::owned` here more, since we are explicit about creating an owned type.

Reply:
I think we should remove `new_for_test` - I checked out and don't see any usages of this function anymore (replaced with `CS::owned` and `CS::borrowed` calls in tests).